adal/oauth2_client.py

```diff
@@ -168,7 +168,7 @@ def _validate_token_response(self, body):
             wire_response = json.loads(body)
         except ValueError:
             self._log.exception(
-                'The token response from the server is unparseable as JSON: %(token_response)s',
+                'The token response from the server is unparsable as JSON: %(token_response)s',

```

Comment:
Can you educate me on what data source you use to decide which one is the correct spelling? The "unparseable/unparsable" presumably became popular only after the computer/internet age. Neither of them can be found from for example https://dictionary.cambridge.org/spellcheck/english/?q=unparsable or https://www.merriam-webster.com/dictionary/unparsable

If there is no decisive evidence to go one way or another, I'd suggest keeping the current spelling as-is, so that there would be less variations to be missed out in a future troubleshooting.

Reply:
Noted. Recommendation by https://github.com/codespell-project/codespell.

I think this is similar to [useable vs. useable](https://english.stackexchange.com/questions/69709/is-useable-preferred-in-certain-regions-or-just-an-alternate-spelling-of-usa).

> I'd suggest keeping the current spelling as-is, so that there would be less variations to be missed out in a future troubleshooting.

Agreed. I will revert this change.